src/brand.md

```diff
@@ -119,4 +115,9 @@ below are specific exceptions to the rules described above:
    Where the Flutter trademarks are displayed on a social media account
    as part of a community site name, you should include a disclaimer that
    clarifies that it is not an official Google account.
+   
+3. <b>[Unofficial Flutter Events]</b> You may use the Flutter trademarks as part of the name 
+   of a community event (e.g. conference), but please make sure to include the following disclaimer 
```

Comment:
e.g. => for example, a

We don’t use e.g.: https://developers.google.com/style/abbreviations#dont-use

Also, we break our source lines up at 80 chars or less. These lines seem longer.